clusterloader2/testing/load/modules/measurements.yaml

```diff
@@ -151,6 +151,21 @@ steps:
       - name: Perc50
         query: quantile_over_time(0.50, sum by (container) (container_memory_working_set_bytes / 1024 / 1024)[%v:])
 {{end}}
+  - Identifier: Quotas total usage
+    Method: GenericPrometheusQuery
+    Params:
+      action: {{$action}}
+      metricName: Quota usage
+      metricVersion: v1
+      prometheusClient: managed
+      unit: Queries
+      queries:
+      - name: default
+        query: sum by (quota_metric) (delta(serviceruntime_googleapis_com:quota_rate_net_usage{monitored_resource="consumer_quota", quota_metric="compute.googleapis.com/default"}[%v]))
+      - name: list_requests
+        query: sum by (quota_metric) (delta(serviceruntime_googleapis_com:quota_rate_net_usage{monitored_resource="consumer_quota", quota_metric="compute.googleapis.com/list_requests"}[%v]))
+      - name: read_requests
+        query: sum by (quota_metric) (delta(serviceruntime_googleapis_com:quota_rate_net_usage{monitored_resource="consumer_quota", quota_metric="compute.googleapis.com/read_requests"}[%v]))
```

Comment:
For testing, this metric is fine, but I would prefer not to relay on it as values will depend on length of test etc.

Reply:
Done.